### PR TITLE
[API] Add timeout mode to gstTest command

### PR DESCRIPTION
**Pull request by dongju-chae**
This patch adds timeout mode to gstTest command.
It prevents from possible hanging problems.

This is a rough proposal and please share any better solutions.

CC: @gichan-jang 

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

### CODE REVIEW

**Review by gichan-jang — APPROVED** [review]
It's a great change. I needed this!! Thanks.

File: debian/rules [hunk 1]
```diff
@@ -16,5 +16,7 @@
 %:
 	dh $@
 
+override_dh_auto_clean:
+	rm -f ssat
```
Comment (myungjoo) [inline debian/rules:20]: is this relevant?

**Review by myungjoo — APPROVED** [review]
LGTM other than the change in /debian/rules that I don't understand. I will wait for the answer before merging this :)

File: debian/rules (hunk 1 above)
Comment (dongju-chae) [reply debian/rules:20]: Ah it's not relevant to this PR but minor patch to clean files generated by the previous build.

**Pull request merged**